app/controllers/recipients_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RecipientsController < ApplicationController
```

Comment:
Not for the matching function ... any logged-in user can do it.  We may want to rethink that later - a member should probably only get to match themselves with people in their campaign_org.  But there's no incentive for a member to do so ... and I don't think we should complicate matters.